docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -483,6 +495,7 @@ defined by the longest pulse Vertiq modules accept as PWM throttles, 2000us.
 
 In the module configuration completed so far in this tutorial, there is no configured :ref:`disarming region <disarm_on_throttle>`. So, we will set our Disarmed output to 1000us and our 
 Minimum output to 1005us. Note that our Disarmed output should always be lower than the Minimum output. The Maximum value can remain 2000us.
+**It is highly recommended to enable the Disarm On Throttle parameter through Control Center and configure your disarming region for safety purposes.**
```

Comment:
We can't just say this when we haven't covered how to set this up in the rest of the tutorial. If you want to tell people to do this, we need to update the tutorial to actually tell people how to set this up from the beginning. Otherwise, this shouldn't be here as it basically says "All that module configuration we covered earlier is wrong and unsafe". Because we never actually set up a disarming region in the module configuration parts above. We just briefly mention its possibility in the random paragraph example above. Which I think @vertiq-jordan is actually moving in his extant PR, so you guys may want to coordinate on that